tests/apps/actorapp/go.mod

```diff
@@ -0,0 +1,7 @@
+module app
+
+go 1.14
+
+require github.com/dapr/dapr v0.8.1-0.20200617162410-8cf5820af38f
+
+replace k8s.io/client => github.com/kubernetes-client/go v0.0.0-20190928040339-c757968c4c36
```

Comment:
Do we need this ?

Reply:
Yes, without the replace statement, go mod tidy shows errors:-
go: github.com/dapr/dapr@v0.8.0 requires
    github.com/dapr/components-contrib@v0.1.0 requires
    github.com/dapr/dapr@v0.4.1-0.20200228055659-71892bc0111e requires
    github.com/dapr/components-contrib@v0.0.0-20200219164914-5b75f4d0fbc6 requires
    k8s.io/client@v0.0.0-00010101000000-000000000000: invalid version: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /Users/vinayada/go/pkg/mod/cache/vcs/1873572271fcf325ad32062ccf6aee165fdb6b2229b714b40050d6bf3d599824: exit status 128:
    remote: Repository not found.
    fatal: repository 'https://github.com/kubernetes/client/' not found
~/dapr/dapr/tests/apps/service_invocation % go get -u k8s.io/client@v0.0.0-00010101000000-000000000000
go get k8s.io/client@v0.0.0-00010101000000-000000000000: k8s.io/client@v0.0.0-00010101000000-000000000000: invalid version: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /Users/vinayada/go/pkg/mod/cache/vcs/1873572271fcf325ad32062ccf6aee165fdb6b2229b714b40050d6bf3d599824: exit status 128:
    remote: Repository not found.
    fatal: repository 'https://github.com/kubernetes/client/' not found